SparkPost/2web2ui
# FE-960: Hibana style cleanup for snippets pages

### What Changed
 - Touched up Snippet pages to work with both OG and Hibana
 - Added a new button variant, "connected" and used it everywhere

### How To Test
 - Confirm Snippet list, create, and edit pages looks good in both OG and Hibanan
